### PR TITLE
397 message template screen

### PR DESCRIPTION
**Pull request by gborgonia-ir**
Ticket ID: #397

### CODE REVIEW

File: src/components/ui/icon/icons/message-icon.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+export default function GanttChartIcon({ size }: { size: number }) {
```
Comment (jatnerubia) [inline src/components/ui/icon/icons/message-icon.tsx:1]: typo

**Review by jatnerubia — CHANGES_REQUESTED** [review]
^^

File: src/features/admin/email-templates/email-templates-filter.tsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  label: value,
+  value,
+}))
+
+export const EmailTemplatesFilter = () => {
+  const [searchParams, setSearchParams] = useSearchParams()
+
+  const [name, setName] = useState<string>(searchParams.get("name") ?? "")
+  const [template_type, setTemplateType] = useState<string>(searchParams.get("template_type") ?? "")
+  const [is_default, setDefault] = useState<string>(searchParams.get("defult") ?? "all")
```
Comment (emmaacha) [inline src/features/admin/email-templates/email-templates-filter.tsx:30]: Typo for `default`

**Review by emmaacha — APPROVED** [review]

**Review by emmaacha — CHANGES_REQUESTED** [review]
Kindly fix typo, misclicked approve for this change request 😅

**Review by jatnerubia — APPROVED** [review]
Cool

**Review by emmaacha — APPROVED** [review]

**Pull request merged**